### PR TITLE
Add Zulu JDK 7.0.171

### PR DESCRIPTION
**Pull request by vpavic**

### CODE REVIEW

File: src/main/scala/io/sdkman/changelogs/JavaMigrations.scala [hunk 1]
```diff
@@ -47,4 +47,13 @@ class JavaMigrations {
       Version("java", "9.0.4-openjdk", "https://download.java.net/java/GA/jdk9/9.0.4/binaries/openjdk-9.0.4_osx-x64_bin.tar.gz", MacOSX),
       Version("java", "10.0.0-openjdk", "https://download.java.net/java/GA/jdk10/10/binaries/openjdk-10_osx-x64_bin.tar.gz", MacOSX))
   }
+
+  @ChangeSet(order = "007", id = "007-add_zulu_7_0_171", author = "vpavic")
```
Comment (marc0der) [inline src/main/scala/io/sdkman/changelogs/JavaMigrations.scala]: As part of this PR can we also clean up the residual 7.0.141-zulu versions please?

**Comment by vpavic** [comment]
> As part of this PR can we also clean up the residual 7.0.141-zulu versions please?

Done.

**Comment by marc0der** [comment]
This seems to be conflicting with your previous PR now, sorry! Could you rebase against master and make any appropriate changes please?

In future, it might be easier to do all such changes in a single PR. wdyt?

**Comment by marc0der** [comment]
Thank you, that's perfect now. Thanks again for contributing!

**Pull request merged**